### DISCUSSION TITLE
Combine open/change channel dialog with select filters dialog

### DISCUSSION THREAD

**Discussion by embrsword** [Chatterino/chatterino2#4528] [Ideas]
I usually create two splits for any sizable channels, one with filters applied and one thats the firehose

Adding the filter when creating the split would also apply it to the chat history before it is loaded in and avoid a big dump of unfiltered chat coming through as well as simplifying the UI

**Comment by Felanbird** [comment]
As a minor workaround, you can apply the filter and then `CTRL+W` both tabs and `CTRL+G` them to bring them back, and the restored tabs will load with any applied filters.
